packages/backend/auth/src/generate-apple-client-secret.ts

```diff
@@ -0,0 +1,35 @@
+import * as jwt from "jsonwebtoken";
+
+interface AppleClientSecretGeneratorParams {
+  teamId: string;
+  clientId: string;
+  keyId: string;
+  appleKey: string;
+}
+
+export const generateAppleClientSecret = (params: AppleClientSecretGeneratorParams) => {
```

Comment:
Would LOVE a test written for this function but can wait if that's going to take too long

Reply:
Agreed - but keen to get this used on Qrone so I'll merge this now and open an issue here to add tests?